rel/files/app.config

```diff
@@ -1,9 +1,51 @@
 [
  {kernel, [
-  {logger_level, warning},
+  {logger_level, info},
+
   {logger, [
+    {filters, log, [
+           %% If we want to see complete accumulator in logs
+        %  {preserve_acc_filter, {fun mongoose_log_filter:preserve_acc_filter/2, no_state}},
+           {format_packet_filter, {fun mongoose_log_filter:format_packet_filter/2, no_state}},
+           {format_acc_filter, {fun mongoose_log_filter:format_acc_filter/2, no_state}},
+           {format_c2s_state_filter, {fun mongoose_log_filter:format_c2s_state_filter/2, no_state}},
+           {format_stacktrace_filter, {fun mongoose_log_filter:format_stacktrace_filter/2, no_state}}
+        ]},
+
     %% Console logger
-    {handler, default, logger_std_h, #{}},
+%   {handler, otp_shell_log, logger_std_h, #{
+%        formatter => {logger_formatter, #{
+%          depth => 12,
+%          chars_limit => 4096
+%     }}
+%   }},
+    {handler, shell_log, logger_std_h, #{
+         filters => [
+           %% Extracts fields, used in the template
+           {fill_metadata_filter, {fun mongoose_log_filter:fill_metadata_filter/2,
+                                   [what, text, user, from_jid, to_jid,
+                                    class, reason, stacktrace]}}
+         ],
+         formatter => {flatlog, #{
+           map_depth => 3,
+           term_depth => 50,
+
+           %% Template with custom fields
+           template => [colored_start, "when=", time, " level=", level,
```

Comment:
so, if we want to configure two flatlogs, one for shell, one for files, we would need to copy this monstrosity.

Reply:
I'd make this template more readable by strictly separating one key per line, even if this means having too many lines; or by indenting lines in columns.
And yeah, I'd be good to make it a variable in some upper level place where it can be used in multiple places. A global or whatever.

Reply:
You could also create a new module like `mongoose_flatlog_formatter` which would just call `flatlog` with these defaults that work for MIM embedded (they could of course be overridable as well).

Reply:
oh, awesome idea :)